### PR TITLE
Demos for list-toolbar, list-paging, and list-filters component

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

**Review by Blackbaud-SteveBrush — DISMISSED** [review]

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-list-builder?_sp.spa=pr-demos](https://host.nxt.blackbaud.com/skyux-list-builder?_sp.spa=pr-demos)

File: src/app/docs/list-filters/demo/list-filters-demo-modal.component.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
+<sky-modal>
+  <sky-modal-header>
+    {{headerText}}
+  </sky-modal-header>
+  <sky-modal-content>
+    <label
+      [for]="fruitTypeSelect.id"
+    >
```
Comment (blackbaud-johnly) [inline src/app/docs/list-filters/demo/list-filters-demo-modal.component.html:8]:
```suggestion
      class="sky-control-label"
    >
```
The label seems too tight to the input.

File: src/app/docs/list-filters/demo/list-filters-demo-modal.component.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      </sky-checkbox>
+    </div>
+  </sky-modal-content>
+  <sky-modal-footer>
+    <button
+      class="sky-btn sky-btn-primary"
+      type="button"
+      (click)="applyFilters()"
+    >
+      Apply filters
```
Comment (blackbaud-johnly) [inline src/app/docs/list-filters/demo/list-filters-demo-modal.component.html:51]:
```suggestion
      Apply
```

File: src/app/public/plugin-resources/code-examples/list-filters/modal-filters/list-filters-demo-modal.component.html [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+      </sky-checkbox>
+    </div>
+  </sky-modal-content>
+  <sky-modal-footer>
+    <button
+      class="sky-btn sky-btn-primary"
+      type="button"
+      (click)="applyFilters()"
+    >
+      Apply filters
```
Comment (blackbaud-johnly) [inline src/app/public/plugin-resources/code-examples/list-filters/modal-filters/list-filters-demo-modal.component.html:51]:
```suggestion
      Apply
```

File: src/app/public/plugin-resources/code-examples/list-filters/modal-filters/list-filters-demo-modal.component.html [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    </button>
+    <button
+      class="sky-btn sky-btn-link"
+      type="button"
+      (click)="clearAllFilters()"
+    >
+      Clear all filters
```
Comment (blackbaud-johnly) [inline src/app/public/plugin-resources/code-examples/list-filters/modal-filters/list-filters-demo-modal.component.html:58]:
```suggestion
      Clear filters
```

File: src/app/docs/list-filters/demo/list-filters-demo-modal.component.html [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+    </button>
+    <button
+      class="sky-btn sky-btn-link"
+      type="button"
+      (click)="clearAllFilters()"
+    >
+      Clear all filters
```
Comment (blackbaud-johnly) [inline src/app/docs/list-filters/demo/list-filters-demo-modal.component.html:58]:
```suggestion
      Clear filters
```

File: src/app/public/plugin-resources/code-examples/list-filters/inline-filters/list-filters-demo.component.html [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+        defaultValue="any"
+        name="fruitType"
+        value="any"
+        [filter]="fruitTypeFilterFunction"
+      >
+        <ng-template
+          let-filter="filter"
+        >
+          <label
+            [for]="fruitTypeSelect.id"
```
Comment (blackbaud-johnly) [inline src/app/public/plugin-resources/code-examples/list-filters/inline-filters/list-filters-demo.component.html:16]:
```suggestion
            [for]="fruitTypeSelect.id"
            class="sky-control-label"
```

File: src/app/public/plugin-resources/code-examples/list-filters/modal-filters/list-filters-demo-modal.component.html [hunk 7]
```diff
@@ -0,0 +1,68 @@
+<sky-modal>
+  <sky-modal-header>
+    {{headerText}}
+  </sky-modal-header>
+  <sky-modal-content>
+    <label
+      [for]="fruitTypeSelect.id"
```
Comment (blackbaud-johnly) [inline src/app/public/plugin-resources/code-examples/list-filters/modal-filters/list-filters-demo-modal.component.html:7]:
```suggestion
      [for]="fruitTypeSelect.id"
      class="sky-control-label"
```

File: src/app/public/plugin-resources/code-examples/list-filters/modal-filters/list-filters-demo-modal.component.ts [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+import {
+  SkyModalInstance
+} from '@skyux/modals';
+
+import {
+  ListFiltersDemoModalContext
+} from './list-filters-demo-modal-context';
+
+@Component({
+  selector: 'sky-demo-filter-modal-form',
```
Comment (blackbaud-johnly) [inline src/app/public/plugin-resources/code-examples/list-filters/modal-filters/list-filters-demo-modal.component.ts:22]:
```suggestion
  selector: 'app-demo-filter-modal-form',
```

File: src/app/docs/list-filters/demo/list-filters-docs-modal.component.ts [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+import {
+  SkyModalInstance
+} from '@skyux/modals';
+
+import {
+  ListFiltersDocsModalContext
+} from './list-filters-docs-modal-context';
+
+@Component({
+  selector: 'sky-demo-filter-modal-form',
```
Comment (blackbaud-johnly) [inline src/app/docs/list-filters/demo/list-filters-docs-modal.component.ts:22]:
```suggestion
  selector: 'app-demo-filter-modal-form',
```

**Review by blackbaud-johnly — COMMENTED** [review]
A few suggested tweaks. ... Also, the toolbar docs are no longer accessible, so I haven't checked updates there. Did the path change or something?

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-list-builder?_sp.spa=pr-demos](https://host.nxt.blackbaud.com/skyux-list-builder?_sp.spa=pr-demos)

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**